app/src/main/res/layout/download_menu_cell.xml

```diff
@@ -3,7 +3,7 @@
     xmlns:android="http://schemas.android.com/apk/res/android"
     android:layout_width="match_parent"
     android:layout_height="56dp"
-    android:background="@android:color/white"
+    android:background="?android:attr/selectableItemBackground"
```

Comment:
We actually have a `@drawable/round_rectangle_ripple`, let's use this and also use this for Zerda/app/src/main/res/layout/item_history_website.xml and /Users/tyu/Documents/zerda/Zerda/app/src/main/res/layout/download_menu_cell.xml too. Let @idiotmax know if we need other tickets for this.